README.rst

```diff
@@ -74,7 +74,8 @@ a peer reviewed paper introducing the library (22 authors).
 Installation
 ------------
 
-The library requires Python 3.5 or greater.
+The library requires Python 3.6 or greater. We typically support the two latest
```

Comment:
I know we said that we'd support the two latest in the past but perhaps we keep previous versions until there's a particular feature we want to take advantage of? (Perhaps there already is and I'm forgetting.)

Reply:
I've just seen https://github.com/Axelrod-Python/Axelrod/pull/1233#issuecomment-453702661 which seems like a good reason to me. Would it be ok to just bump here to 3.6 and not include the `We typically support the two latest...`?